README.md

```diff
@@ -2,21 +2,63 @@
 
 # prom-label-proxy
 
-The prom-label-proxy can enforce a given label in a given PromQL query, in Prometheus API responses or in Alertmanager API requests.
+The prom-label-proxy can enforce a given label in a given PromQL query, in Prometheus API responses or in Alertmanager API requests. This allows read multi-tenancy for projects like Prometheus, Alertmanager or Thanos.
 
-This proxy does not perform authentication or authorization, this has to happen before the request reaches this proxy. The [kube-rbac-proxy](https://github.com/brancz/kube-rbac-proxy) is an example for such an additional building block.
+This proxy does not perform authentication or authorization, this has to happen before the request reaches this proxy, allowing you to use any auth* system you want. The [kube-rbac-proxy](https://github.com/brancz/kube-rbac-proxy) is an example for such an additional building block.
 
-Risks outside the scope of this project:
+### Risks outside the scope of this project:
 
-- If a tenant controls its scrape target configuration the tenant can set arbitrary labels via its relabelling configuration, thereby being able to pollute other tenant's metrics.
+It's not a goal for this project to solve write tenant isolation for multi-tenant Prometheus:
 
-- If the ingestion configuration [honor_labels](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#scrape_config) is set for a tenant's target, that target can pollute other tenant's metrics as Prometheus respects any labels exposed by the target.
+* If a tenant controls its scrape target configuration the tenant can set arbitrary labels via its relabelling configuration, thereby being able to pollute other tenant's metrics.
+* If the ingestion configuration [honor_labels](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#scrape_config) is set for a tenant's target, that target can pollute other tenant's metrics as Prometheus respects any labels exposed by the target.
+
+See [Thanos soft/hard tenancy](https://thanos.io/tip/proposals/201812_thanos-remote-receive.md/#architecture) or [Cortex](https://cortexmetrics.io/) as example solution to that.
```

Comment:
If we're listing examples, then the Prometheus Operator label enforcement functionality must also be listed, especially as that was the initial purpose of this component.

Reply:
Yes!